### ISSUE TITLE
Not working in browser: Module._malloc is not a function

### ISSUE THREAD

**Issue by philholden** [1p6/supercop.js#1]
http://requirebin.com/?gist=2b2132513fab75bbc27a528a8d69c76d

console error:

```
/tmp/supercop.js116311-30934-kkma7x/node_modules/supercop.js/index.js:10 Uncaught TypeError: Module._malloc is not a function
exports.createKeypair

@ /tmp/supercop.js116311-30934-kkma7x/node_modules/supercop.js/index.js:10(anonymous function) 
@ /tmp/supercop.js116311-30934-kkma7x/node_modules/supercop.js/index.js:68setTimeout (async)(anonymous function)
@ /tmp/supercop.js116311-30934-kkma7x/node_modules/supercop.js/lib.js:3
```

**Comment by philholden** [comment]
`module['exports']` is not being set for web environment:

``` javascript
else if (ENVIRONMENT_IS_WEB || ENVIRONMENT_IS_WORKER) {
  ...
  // I can get it to work by adding 
  if (typeof module !== 'undefined') {
    module['exports'] = Module;
  }
}
```

I have to add this in the `lib.js` line 165 not sure how to fix more permanently as lib is auto generated.

**Comment by philholden** [comment]
Actually you can just append at the end of lib.js in `build.js`:

``` javascript
fs.appendFileSync(
  'lib.js',
  'if (typeof module !== "undefined") {  module["exports"] = Module; }'
);
```

**Comment by 1p6** [comment]
Fixed the bug in version 1.0.2.
